api/v2/conversion.go

```diff
@@ -1,5 +1,5 @@
 //
-// Copyright (c) 2019-2021 Red Hat, Inc.
```

Comment:
I believe the usual naming format for this file is `<crd>_conversion.go`, e.g. `checluster_conversion.go`, in order to match `checluster_types.go` and `checluster_webhook.go`.

I also can't easily find the similar `conversion.go` bits in the v1 apiVersion